src/styles/blocks/starsPlusReviews.scss

```diff
@@ -0,0 +1,42 @@
+.starsPlusReviews {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 100%;
+  line-height: 16px;
+
+  font-size: 10px;
+  margin-bottom: 24px;
+
+  &__prodReviews {
+    color: $main-accent-color;
+    font-size: 10px;
+    width: 53px;
+  }
+}
+
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background-image: url(/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &div:last-child {
+    margin-right: 0;
+    //doesnot work! test wo this margin, fix after or delete
+  }
```

Comment:
Read more about [:last-child](https://developer.mozilla.org/en-US/docs/Web/CSS/:last-child)

Reply:
@Olexandr6  what correction do you expect?